web/src/components/TeamInfo.vue

```diff
@@ -82,6 +82,12 @@ export default defineComponent({
               />
             </span>
           </div>
+          <div
+            v-if="item.homepage_website"
```

Comment:
```suggestion
            v-if="item.homepage_website && item.homepage_website[0]"
```

Should we add this check since empty arrays in javascript are truthy?

Reply:
good catch!